src/Files.App/Data/Commands/HotKey/HotKeyCollection.cs

```diff
@@ -1,50 +1,84 @@
-﻿using Files.App.Actions;
-using System;
-using System.Collections;
-using System.Collections.Generic;
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
 using System.Collections.Immutable;
-using System.Diagnostics;
-using System.Linq;
 
 namespace Files.App.Data.Commands
 {
+	/// <summary>
+	/// Represents collection of <see cref="HotKey"/>.
+	/// </summary>
 	[DebuggerDisplay("{Code}")]
 	public readonly struct HotKeyCollection : IEnumerable<HotKey>, IEquatable<HotKeyCollection>
 	{
-		private static readonly char[] parseSeparators = new char[] { ',', ';', ' ', '\t' };
+		// Fields
+
+		private static readonly char[] parseSeparators = [' ', '\t'];
```

Comment:
Don't we want `,` to be a valid separator?

Reply:
',' is also valid hotkey. Using ',' as separator causes to misinterpret. Space or Tab is only valid for this situation because  we store them as 'Space' or 'Tab' not actual  chat